initialization/initialization.go

```diff
@@ -298,6 +310,25 @@ func (init *Initializer) Initialize(ctx context.Context) error {
 	return fmt.Errorf("no nonce found")
 }
 
+func (init *Initializer) removeRedundantFiles(layout config.FilesLayout) error {
+	numFiles, err := init.diskState.NumFiles()
+	if err != nil {
+		return err
+	}
+
+	diff := numFiles - int(layout.NumFiles)
+	for i := 0; i < diff; i++ {
+		index := numFiles - 1 - i
```

Comment:
Nit, how about:
```suggestion
	for i := numFiles-1 ; i >= layout.NumFiles; i-- {
```

Reply:
Even simpler imo:

```suggestion
	for i := layout.NumFiles; i < numFiles; i++ {
```

Reply:
Yes, wanted to do this but forgot ;) 

Fixed.